Examples/Modules/RigidInjection/inputs_2d_BoostedFrame

```diff
@@ -69,6 +69,7 @@ btd_pltfile.dt_snapshots_lab = 1.8679589331096515e-13
 btd_pltfile.fields_to_plot = Ex Ey Ez Bx By Bz jx jy jz rho
 btd_pltfile.format = plotfile
 btd_pltfile.buffer_size = 32
+btd_pltfile.write_species = 1
```

Comment:
Can be a follow-up PR:
Should this file also add `btd_openpmd.write_species = 1`?

Reply:
Thank you @ax3l 
`write_species =1 ` is on by default. 
Actually it's not really needed here, since 1 the default.
I was turning it off (0) and on (1) when testing locally and when plotfile passed for particles. 
When plotfile passed with particles, I left it in there.